tutorials/appstudio-onboarding/appstudio-onboarding.md

```diff
@@ -117,13 +121,17 @@ In this step, you will add the SAP Business Application Studio subscription to a
 
     !![legalterms](2020-03 AppStudio Terms__.jpg)
 
-6. A new tab opens and SAP Business Application Studio loads.
+6. A new tab opens and the welcome page for SAP Business Application Studio loads.
+
+    !![sapbusinessapplicationstudiowelcome](BAS_Welcome_.png)
 
+7. Click **My Dev Spaces** to open the SAP Business Application Studio dev space manager page.
+    <br>
     >**Bookmark this page!**
 
     >If you create a bookmark to this page, it is easy to get back to SAP Business Application Studio later.
 
-    !![sapbusinessapplicationstudioloaded](2020-03 AppStudio Loaded_.jpg)
+    !![sapbusinessapplicationstudio dev space manager](BAS Dev Space Manager Empty_.png)
```

Comment:
Please follow the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64) when naming your images.